docs/current_docs/api/module-dependencies.mdx

```diff
@@ -86,6 +97,8 @@ Next, create a new Dagger Function:
 ```typescript file=./snippets/dependencies/chaining/typescript/index.ts
 ```
 </TabItem>
+```php file=./snippets/dependencies/chaining/php/src/MyModule.php
```

Comment:
@charjr I think you missed the `<TabItem>...</TabItem>` here.

Error from CI: `Error: Docusaurus error: Bad <Tabs> child <MDXPre>: all children of the <Tabs> component should be <TabItem>, and every <TabItem> should have a unique "value" prop.`